### PR TITLE
MMD example enhancement

### PR DESCRIPTION
**Pull request by takahirox**
This PR enhances `webgl_loader_mmd` example.
User can turn on/off each MMD behavior
and can see what techniques MMD uses.

Let me turn off  show IK bones and show rigid bodies options by default
because they slow the performance a little bit.

### CODE REVIEW

File: examples/webgl_loader_mmd.html [hunk 1]
```diff
@@ -163,34 +231,25 @@
...
-				camera.position.y += ( - mouseY - camera.position.y ) * .05;
-
-				camera.lookAt( scene.position );
-
 				if ( mesh ) {
 
 					helper.animate( clock.getDelta() );
-					if ( physicsHelper !== undefined ) physicsHelper.update();
-					ikHelper.update();
+					if ( physicsHelper !== undefined && physicsHelper.visible ) physicsHelper.update();
```
Comment (mrdoob) [inline examples/webgl_loader_mmd.html]: Shouldn't this just be...

```js
if ( physicsHelper.visible ) physicsHelper.update();
```

**Comment by mrdoob** [comment]
I think I would also remove the `isMobileDevice` stuff and enable everything by default. Mobiles are getting pretty fast!

File: examples/webgl_loader_mmd.html (hunk 1 above)
Comment (takahirox) [inline examples/webgl_loader_mmd.html]: Oops, thanks!

**Comment by takahirox** [comment]
OK, I've removed!
(Though physics is still p heavy on my iPod touch 5 haha)

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**